hardware/opas/TOPAS/TOPAS.py

```diff
@@ -210,59 +142,21 @@ def close(self):
         self.api.close()
 
     def get_motor_positions(self):
-        for i in range(6):
-            motor_mutex = list(self.motor_positions.values())[i]
-            error, position_steps = self.api.get_motor_position(i)
-            error, position = self.api.convert_position_to_units(i, position_steps)
+        for m, motor_mutex in self.motor_positions.items():
+            position = self.api.get_motor_position(m)
             motor_mutex.write(position)
         if self.poynting_correction:
             self.poynting_correction.get_motor_positions()
     
-    def get_speed_parameters(self, inputs):
```

Comment:
Speed parameter stuff was only used for homing, since that is gone, I have removed it here.
I do not believe the REST API provides for changing speed parameters (though you can retrieve them)